CODEOWNERS

```diff
@@ -10,6 +10,9 @@
 # File patterns work mostly like .gitignore. Try to keep this file
 # simple because it's literally impossible to test.
 
+# installer
+installer/**				@marc-hb
```

Comment:
OK but can you move it closer to the CMakeLists.txt rules below? Just before them I guess.

Reply:
ALso done.